src/cookie.rs

```diff
@@ -64,12 +66,12 @@ impl<'a> Cookie<'a> {
     pub fn max_age(&self) -> Option<std::time::Duration> {
         self.0
             .max_age()
-            .map(|d| std::time::Duration::new(d.num_seconds() as u64, 0))
+            .map(|d| d.try_into().unwrap())
```

Comment:
Should this unwrap? If this *should* never fail, we can use `expect` with a string describing why. Otherwise, I suppose we can just use `.and_then(|d| d.try_into().ok())`.

Reply:
I changed it to use `expect`, looking at the max-age parsing code it doesn't look like time::Duration will ever overflow the conversion to std::time::Duration
https://github.com/SergioBenitez/cookie-rs/blob/f7f6e741521c866bf9b6c290b4389425b5c952e7/src/parse.rs#L180-L197